data/actions/scripts/other/explosive_present.lua

```diff
@@ -0,0 +1,6 @@
+function onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	player:say("KABOOOOOOOOOOM!", TALKTYPE_MONSTER_SAY)
+	player:getPosition():sendMagicEffect(CONST_ME_EXPLOSIONHIT)
+	item:remove()
+	return true
+end
```

Comment:
always add a new line in the end of new files, and in this one add the achievement as well :P